src/ApiTestCase.php

```diff
@@ -87,6 +92,12 @@ public function setUpDatabase()
         }
     }
 
+    public function setUp()
+    {
+        parent::setUp();
+        $this->fixtureLoader = new Loader();
```

Comment:
:+1: for `setUpLoader`, it has nothing to do with the database as we can use it with InMemoryRepository also.